Automattic/jetpack
# Boost: Remove lazy-loading

Fixes #33953

## Proposed changes:
* Remove lazy-loading module
* Remove it's references from test, readme, etc.

### Other information:

## Jetpack product discussion
No

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
* Test to make sure nothing breaks
* Make sure lazy-loading is no longer available
